representatives/templatetags/representatives_tags.py

```diff
@@ -0,0 +1,21 @@
+from django import template
+from django.utils.safestring import mark_safe
+from django.contrib.humanize.templatetags.humanize import naturalday
+
+register = template.Library()
+
+
+@register.filter
+def country_flag(country):
+    return mark_safe(
+        '<span class="flag-icon flag-icon-{code}"></span> {name}'.format(
+            name=country.name,
+            code=country.code.lower()))
+
+
+@register.filter
+def mandate_date(date, arg=None):
+    if date.year == 9999:
```

Comment:
You can't set a datetime's year to None !

In [5]: datetime.date(year=None,month=1,day=1)
TypeError: an integer is required

I dislike this too, but can live with it.